src/script/arch-update.sh

```diff
@@ -195,22 +195,22 @@ icon_dir="/usr/share/icons/${name}"
 
 # Definition of the icon_checking function: Change icon to "checking"
 icon_checking() {
-	cp -f "${icon_dir}/${name}_checking.svg" "${icon_dir}/${name}.svg" || exit 3
+	echo "${name}_checking" > "${state_dir}/state"
```

Comment:
```suggestion
	echo "${name}_checking" > "${state_dir}/current_state"
```
To comply with the "current/local" prefix already used by the other statefile

Reply:
done this now

Reply:
Thanks!